src/pages/Homepage.js

```diff
@@ -0,0 +1,24 @@
+import { Sidebar } from "components";
+import { useLocation } from "react-router-dom";
+import { AllVideosFeed } from "./AllVideosFeed";
+
+export const Homepage = () => {
+  const location = useLocation();
+  return (
+    <>
+      <div className="video-layout row-flex no-wrap flex-start">
+        <Sidebar />
+        <main className="videos-content">
+          {location.pathname === "/" && <AllVideosFeed />}
+
+          {/* NOTE: Will be uncommenting after adding pages */}
```

Comment:
suggestion: if these lines are not required, you can remove them

Reply:
Sure will remove them after adding the required pages.